src/dv360-utils.ts

```diff
@@ -0,0 +1,38 @@
+/**
+    Copyright 2023 Google LLC
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        https://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+export type logicalOperation = 'OR'|'AND';
```

Comment:
Please Pascal-case the type to LogicalOperation as per https://ts.dev/style/#identifiers

Reply:
This is a logical operation in the filter in DV360 API call (so I would keep it so): https://developers.google.com/display-video/api/reference/rest/v2/advertisers.campaigns/list
```
All ENTITY_STATUS_ACTIVE or ENTITY_STATUS_PAUSED campaigns under an advertiser: (entityStatus="ENTITY_STATUS_ACTIVE" OR entityStatus="ENTITY_STATUS_PAUSED")
```

Reply:
Hi Andriy, I was not talking about the values but the current name of the TS type "logicalOperation". Type identifiers should be Pascal-cased and that should not interfere with the API. So "logicalOperation" should be "LogicalOperation".

Reply:
May request refers to the current name of the TS type "logicalOperation". Type identifiers should be Pascal-cased and that should not interfere with the DV360 API. So "logicalOperation" should be "LogicalOperation".

Reply:
Sorry, for misunderstanding. I've pushed the changes